apps/style/curriculum/navigation.scss

```diff
@@ -28,6 +28,14 @@
       color: $white;
     }
 
+    .doc-links {
```

Comment:
could this be moved somewhere else, or maybe change the `.open` rule to to set `display: none` instead of `height: auto`? I think if we want the nav bar to be generic it shouldn't mention 'docs'

Reply:
Ah good call on the not using "docs" here -- I'll change that. I'm not sure I understand your suggest though. If the category is open, we wouldn't want it to have `display: none` right?

Reply:
er, sorry, right, the opposite of that. I changed what you originally had in the nav bar from `display: block` to `height: auto` to try and get the fancy slide transition that the CB docs do, but it didn't end up working. both approaches should behave the same except that I assume you added back the `display: block` to get keyboard control to work better?

Reply:
Actually, maybe the approach is to just put the `display` rules on the `nav-link` class

Reply:
Yes, the `display: block` was added so that I could use `display: none` when it was closed, which made the keyboard navigation work better. If there's another way to do this let me know!

I didn't put display on the `nav-link` class only because I'd have to pass whether the category was open to `NavigationItem` (I think). But I'm still learning this way of styling so maybe there's a way around that...

Reply:
Yeah it was super easy to move it to `nav-link`. So I'll do that :)